R/git.R

```diff
@@ -323,85 +323,117 @@ git_clean <- function() {
-  hd_line("GitHub")
   default_gh_host <- get_hosturl(default_api_url())
-  kv_line("Default GitHub host", default_gh_host)
-  pat_sitrep(default_gh_host)
+  if ("github" %in% tool && "global" %in% scope) {
+    cli::cli_h3("GitHub (global)")
+    kv_line("Default GitHub host", default_gh_host)
+    pat_sitrep(default_gh_host)
+  }
 
   # git and github for active project ------------------------------------------
-  hd_line("Git repo for current project")
+  if ("project" %in% scope) {
 
-  if (!proj_active()) {
-    ui_info("No active usethis project")
-    return(invisible())
-  }
-  kv_line("Active usethis project", proj_get())
-  if (!uses_git()) {
-    ui_info("Active project is not a Git repo")
-    return(invisible())
-  }
+    if (!proj_active()) {
+      ui_info("No active usethis project")
+      return(invisible())
+    }
+    kv_line("Active usethis project", proj_get())
+    if (!uses_git()) {
+      ui_info("Active project is not a Git repo")
+      return(invisible())
+    }
+
+    # current branch -------------------------------------------------------------
+    branch <- tryCatch(git_branch(), error = function(e) NULL)
+    tracking_branch <- if (is.null(branch)) NA_character_ else git_branch_tracking()
+    # TODO: can't really express with kv_line() helper
+    branch <- if (is.null(branch)) "<unset>" else branch
+    tracking_branch <- if (is.na(tracking_branch)) "<unset>" else tracking_branch
+
+    # local git config -----------------------------------------------------------
+    if ("git" %in% tool) {
+      cli::cli_h3("Git (project)")
+      if (proj_active() && uses_git()) {
+        local_user <- list(
+          user.name = git_cfg_get("user.name", "local"),
+          user.email = git_cfg_get("user.email", "local")
+        )
+        if (!is.null(local_user$user.name) || !is.null(local_user$user.email)) {
+          ui_info("This repo has a locally configured user")
+        }
+        if (!is.null(local_user$user.name)) {
+          kv_line("Name", local_user$user.name)
+        }
+        if (!is.null(local_user$user.email)) {
+          kv_line("Email", local_user$user.email)
+        }
+      }
+
+      # default branch -------------------------------------------------------------
+      default_branch_sitrep()
 
-  # local git config -----------------------------------------------------------
-  if (proj_active() && uses_git()) {
-    local_user <- list(
-      user.name = git_cfg_get("user.name", "local"),
-      user.email = git_cfg_get("user.email", "local")
-    )
-    if (!is.null(local_user$user.name) || !is.null(local_user$user.name)) {
-      ui_info("This repo has a locally configured user")
-      kv_line("Name", local_user$user.name)
-      kv_line("Email", local_user$user.email)
+      # vertical alignment would make this nicer, but probably not worth it
+      ui_bullet(glue("
+      Current local branch -> remote tracking branch:
+      {ui_value(branch)} -> {ui_value(tracking_branch)}"))
```

Comment:
```suggestion
        Current local branch -> remote tracking branch:
        {ui_value(branch)} -> {ui_value(tracking_branch)}"))
```

Closer to the indentation style used elsewhere.

Reply:
corrected in 9fd633a